src/main/java/net/minecraftforge/registries/RegistryBuilder.java

```diff
@@ -227,7 +216,24 @@
         return this;
     }
 
-    public IForgeRegistry<T> create()
+    /**
+     * Enables tags for this registry if not already.
+     * All forge registries with wrappers inherently support tags.
+     *
+     * @return this builder
+     * @see RegistryBuilder#hasWrapper()
+     */
+    public RegistryBuilder<T> hasTags()
+    {
+        // Tag system heavily relies on Registry<?> objects, so we need a wrapper for this registry to take advantage
+        this.hasWrapper();
+        return this;
+    }
+
+    /**
+     * Modders: Use {@link NewRegistryEvent#create(RegistryBuilder)} instead
+     */
+    IForgeRegistry<T> create()
```

Comment:
Should this be `@Deprecated`, to discourage developers from using this method?

Reply:
It's already package private. The documentation is a nicety.